src/object1.cc

```diff
@@ -3017,141 +2975,99 @@ bool_ object_out_desc(object_type *o_ptr, FILE *fff, bool_ trim_down, bool_ wait
 			text_out("It makes you invisible.  ");
 		}
 
-		if (o_ptr->tval != TV_TRAPKIT)
+		vn = 0;
+		if (f2 & (TR2_SUST_STR))
 		{
```

Comment:
Same issue here with indentation. Could you restore it to make it easier to see the actual differences?

(We can always fix in a follow-up commit which absolutely only changes indentation.)